### ISSUE TITLE
line map messed when quickly druging zoom bar in Chrome in windows

### ISSUE THREAD

**Issue by warryy** [apache/echarts#15288]
### Version
https://cdn.jsdelivr.net/npm/echarts@4/dist/echarts.min.js?_v_=1611323308745

### Reproduction link
[https://echarts.apache.org/v4/examples/en/editor.html?c=dynamic-data2](https://echarts.apache.org/v4/examples/en/editor.html?c=dynamic-data2)

### Steps to reproduce
paste code to [https://echarts.apache.org/v4/examples/en/editor.html?c=dynamic-data2]

```
function randomData() {
    now = new Date(+now + oneDay);
    value = value + Math.random() * 21 - 10;
    return {
        name: now.toString(),
        value: [
            [now.getFullYear(), now.getMonth() + 1, now.getDate()].join('/'),
            Math.round(value)
        ]
    };
}

var data = [];
var now = +new Date(1997, 9, 3);
var oneDay = 24 * 3600 * 1000;
var value = Math.random() * 1000;
for (var i = 0; i < 1000; i++) {
    data.push(randomData());
}

option = {
    title: {
        text: '动态数据 + 时间坐标轴'
    },
    tooltip: {
        trigger: 'axis',
        formatter: function (params) {
            params = params[0];
            var date = new Date(params.name);
            return date.getDate() + '/' + (date.getMonth() + 1) + '/' + date.getFullYear() + ' : ' + params.value[1];
        },
        axisPointer: {
            animation: false
        }
    },
    xAxis: {
        type: 'time',
        splitLine: {
            show: false
        }
    },
    yAxis: {
        type: 'value',
        boundaryGap: [0, '100%'],
        splitLine: {
            show: false
        }
    },
    series: [{
        name: '模拟数据',
        type: 'line',
        showSymbol: false,
        hoverAnimation: false,
        data: data
    }],
    dataZoom: {
        show: true
    }
};
```
quickly drug zoom bar

### What is expected?
display normally

### What is actually happening?
line map messed

---
Screenshot link: https://z3.ax1x.com/2021/07/04/RfkDHO.png

**Comment by hereisfahad** [comment]
Try setting the throttle to 0.
`dataZoom: {
        show: true,
        throttle: 0
    }`

**Comment by warryy** [comment]
> Try setting the throttle to 0.
> `dataZoom: { show: true, throttle: 0 }`

not work = =

**Comment by pissang** [comment]
@warryy It should have been fixed in 5.0. You can try it again
